### PR TITLE
Adds example workload for FPerfCSCollection

### PR DESCRIPTION
**Pull request by vulder**
Adapts xz repository naming

### CODE REVIEW

File: varats/varats/projects/c_projects/xz.py [hunk 1]
```diff
@@ -81,7 +82,7 @@ class Xz(VProject):
     WORKLOADS = {
         WorkloadSet(WorkloadCategory.EXAMPLE): [
             Command(
-                SourceRoot("xz") / RSBinary("xz"),
+                SourceRoot("xz.git") / RSBinary("xz"),
```
Comment (vulder) [inline varats/varats/projects/c_projects/xz.py]:
```suggestion
                SourceRoot("xz") / RSBinary("xz"),
```

File: varats/varats/projects/c_projects/xz.py [hunk 2]
```diff
@@ -56,7 +57,7 @@ class Xz(VProject):
             PaperConfigSpecificGit(
                 project_name='xz',
                 remote="https://github.com/xz-mirror/xz.git",
-                local="xz",
+                local="xz.git",
```
Comment (vulder) [inline varats/varats/projects/c_projects/xz.py]:
```suggestion
                local="xz",
```

File: varats/varats/projects/c_projects/xz.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 """Project file for xz."""
 import typing as tp
+from pathlib import Path
```
Comment (vulder) [inline varats/varats/projects/c_projects/xz.py]:
```suggestion
```

**Comment by danjujan** [comment]
Tested it. Works as expected.

**Review by danjujan — APPROVED** [review]

**Pull request merged**